Teacher Workout/Utils/Extensions/Text+Styling.swift

```diff
@@ -14,6 +14,20 @@ extension Text {
             .mask(RoundedRectangle(cornerRadius: 50, style: .continuous))
     }
 
+    func secondaryButtonStyle() -> some View {
+        font(Font.custom("Mulish-Regular", size: 15))
+            .fontWeight(.bold)
+            .tracking(1.25)
+            .foregroundColor(Color.accentColor)
+            .textCase(.uppercase)
+            .padding()
+            .frame(maxWidth: .infinity)
```

Comment:
This secondary button style is almost the same as the primary button style, with 1 line difference. Can we extract the common elements into a base button style or something?